app/web/features/communities/events/EventTimeChanger.tsx

```diff
@@ -52,44 +51,13 @@ export default function EventTimeChanger({
   const { date: eventEndDate, time: eventEndTime } =
     splitTimestampToDateAndTime(event?.endTime);
 
-  const handleStartTimeChange = (e: {
-    target: { value: string | number | dayjs.Dayjs | Date | null | undefined };
-  }) => {
-    if (!e.target.value) {
-      setValue("startTime", "", { shouldDirty: true, shouldValidate: true });
-      return;
-    }
-    const newStartTime = dayjs(e.target.value, TIME_FORMAT);
-
-    setValue("startTime", newStartTime.format(TIME_FORMAT), {
-      shouldDirty: true,
-      shouldValidate: true,
-    });
-  };
-
```

Comment:
I think you might still need to `setValue` for the new Timepicker start and end times. I assume that function is passing it up to the parent form component? Similar to what the Datepicker is doing, it has the Controller, but is still setting the value.